[Logstash] Add docs link to monitoring with agent topic in LSR (#9183)
elastic/integrations: packages/logstash/_dev/build/docs/README.md

# Logstash

> The `logstash` package collects metrics and logs related to Logstash.

Author: This still doesn't feel quite right. 

Maybe " The `logstash` package collects metrics and logs related to Logstash performance and processes." ? 

Other ideas are welcomed.

Reviewer: This is the same copy as the [Elasticsearch integration](https://github.com/elastic/integrations/blob/main/packages/elasticsearch/_dev/build/docs/README.md), but I agree that it doesn't sound quite right.

But maybe something like the [Kafka integration](https://github.com/elastic/integrations/blob/main/packages/kafka/_dev/build/docs/README.md)

```suggestion
This integration collects logs and metrics from Logstash instances.
```

(Also, can we use [this](https://github.com/elastic/integrations/blob/main/packages/kibana/_dev/build/docs/README.md#kibana) syntax for Logstash links?)

You can find additional information about monitoring Logstash with the Logstash integration in the **Logstash Reference**: {{ url "monitoring-with-ea" "Monitoring Logstash with Elastic Agent" }}.

> 

Author: I've posted a question to see if this format will work out of the box, or if extra config is required.

Reviewer 2: Your PR is failing with this error:

```
check if README.md is up-to-date failed: generating readme file failed: rendering Readme failed: executing template failed: template: README.md:5:126: executing "README.md" at <url "monitoring-with-ea" "Monitoring Logstash with Elastic Agent">: error calling url: link key not found: monitoring-with-ea
```

Reviewer 2: You need to define the `monitoring-with-ea` key somewhere. Where is the question.

> You can find additional information about monitoring Logstash with the Logstash integration in the **Logstash Reference**: {{ url "logstash-monitoring-ea" "Monitoring Logstash with Elastic Agent" }}.

Reviewer 3: CI failure is related to not being updated the rendered REAME file located at `packages/logstash/docs/README.md`.
https://buildkite.com/elastic/integrations/builds/9119#018dc8c0-06a3-41a7-a2fd-ed1c3f566c0f/329-337

That file could also be generated based on the contents in `_dev/build/docs/README.md` using `elastic-package` with the command:
```shell
cd packages/logstash
elastic-package build
```

> You can find additional information about monitoring Logstash with the Logstash integration in the **Logstash Reference**: [Monitoring Logstash with Elastic Agent](https://www.elastic.co/guide/en/logstash/current/monitoring-with-ea.html). 

Author: ```suggestion
You can find additional information about monitoring Logstash with the Logstash integration in the **Logstash Reference**: {{ url "monitoring-with-ea" "Monitoring Logstash with Elastic Agent" }}.

[Monitoring Logstash with Elastic Agent](https://www.elastic.co/guide/en/logstash/current/monitoring-with-ea.html). 
```
